agreements/project-contribution/fiscal-sponsorship-agreement.md

```diff
@@ -0,0 +1,277 @@
+
+Sponsor’s governing body (the "Council") has determined that acting as the Project’s fiscal sponsor by receiving assets and incurring liabilities identified for the Project Purposes and using them to pursue the Project Purposes will further the mission of the Sponsor and the common business interests of its members.
+
+The Council has approved the creation of a restricted fund to support the Project’s purposes, as detailed in the [Restricted Fund](#restricted-fund-and-variance-power) section.
+
+The Sponsor is a Florida nonprofit corporation exempt from federal income tax under Section 501(c)(6) of the Internal Revenue Code ("IRC").
+
+The individuals listed as Signatories desire to manage the Project on behalf of the Sponsor, subject to the Sponsor’s [discretion and control](#collaborative-management).
+Future project leaders, upon assuming leadership roles, shall be required to acknowledge and agree to the terms of this agreement as a condition of their role. Records of the current Signatories will be jointly maintained alongside asset inventories as defined in the [Terms and Conditions][].
+
+## Agreement
+
+For purposes of this Agreement, the term “Project” refers to the coordinated activities and associated assets aimed at achieving the Project Purposes, under the stewardship of the Signatories.
+
+On the Effective Date, the Sponsor shall assume responsibility for the Project as described in the [Collaborative Management](#collaborative-management) section.
+This arrangement shall remain in effect unless and until terminated as provided under the [Termination](#termination) section of this Agreement.
+
+**The Project is not a separate legal entity and is not an agent of the Sponsor.**
+The Signatories acknowledge that they and the Project do not, and shall not, act as an agent for the Sponsor unless specifically authorized in writing.
+
+The Signatories shall abide by the [Terms and Conditions][], which the Sponsor may amend as described in those Terms.
+
+### Project Management and Activities
+
+From the Effective Date, the following activities shall be conducted in the name of the Sponsor, under its oversight and guidance, as provided in the [Collaborative Management](#collaborative-management) section of this Agreement and in accordance with the Sponsor's [Terms and Conditions][]:
+
+- Community programs
+- Public information work
+- Fundraising events
+- Processing and acknowledgment of cash and non-cash revenue items
+- Accounts payable and receivable
+- Negotiation of leases and contracts
+- Disbursement of Project funds (including grants)
+- Other activities conducted in furtherance of the Project
+
+### Independent Activities
+
+Individuals and organizations working on the Project are permitted to engage in independent activities to promote and support the Project, including:
+
+- Hosting and participating in community events related to the Project
+- Promoting the Project at various events and through different channels
```

Comment:
This includes writing a book about the project?

Reply:
Yes.